xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-docker/src/main/java/org/xwiki/test/docker/internal/junit5/WikiCreator.java

```diff
@@ -37,6 +37,7 @@
 import org.xwiki.rest.internal.ModelFactory;
 import org.xwiki.rest.model.jaxb.JobRequest;
 import org.xwiki.test.docker.junit5.DockerTestException;
+import org.xwiki.test.extension.JobExecutor;
```

Comment:
Is this needed ?

Reply:
Since I moved `JobExecutor` to `xwiki-platform-test-rest`, yes.

Reply:
OK, make sense. I was surprised to see a new import without any corresponding code :)